<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestphaseWidget</class>
 <widget class="QDialog" name="TestphaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Motorplatform 1</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Thrust1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderThrust1">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxThrust1">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Orientation1 in 0.1 deg.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialOrientation1">
        <property name="minimum">
         <number>-1800</number>
        </property>
        <property name="maximum">
         <number>1800</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxOrientation1">
        <property name="minimum">
         <number>-1800</number>
        </property>
        <property name="maximum">
         <number>1800</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Motorplatform 2</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Thrust2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderThrust2">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxThrust2">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Orientation2 in 0.1 deg.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialOrientation2">
        <property name="minimum">
         <number>-10800</number>
        </property>
        <property name="maximum">
         <number>10800</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxOrientation2">
        <property name="minimum">
         <number>-10800</number>
        </property>
        <property name="maximum">
         <number>10800</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Motorplatform 3</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Thrust3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderThrust3">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxThrust3">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Orientation3 in 0.1 deg.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialOrientation3">
        <property name="minimum">
         <number>-10800</number>
        </property>
        <property name="maximum">
         <number>10800</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxOrientation3">
        <property name="minimum">
         <number>-10800</number>
        </property>
        <property name="maximum">
         <number>10800</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Motorplatform 4</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Thrust4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderThrust4">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxThrust4">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Orientation4 in 0.1 deg.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialOrientation4">
        <property name="minimum">
         <number>-10800</number>
        </property>
        <property name="maximum">
         <number>10800</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxOrientation4">
        <property name="minimum">
         <number>-10800</number>
        </property>
        <property name="maximum">
         <number>10800</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="controlButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Activate Engine</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="homingButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>homing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SetZeroButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>set zero</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopallButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>stop all</string>
       </property>
       <property name="iconSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
